ZombieWar/src/main/java/com/csc422/zombiewar/characters/Character.java

```diff
@@ -68,16 +89,46 @@ public int getAttack()
      */
     public void attacks(Character c)
     {
-        c.setHealth((c.getHealth()) - (this.attack));
+      if(weapon!=null){
+        if(landedHit()==true){
+          if((weapon.getClass().getSimpleName().equals("FryingPan")|weapon.getClass().getSimpleName().equals("CrowBar")|weapon.getClass().getSimpleName().equals("Axe"))){
+              this.attack=weapon.getDamage();
+          }else{
+              this.attack+=weapon.getDamage();
```

Comment:
This isn't going to do what you want it to do.

if a character attacks twice, the attack will increment by the damage amount each time the character attack,  And a character will attack for the number of defenders at least once (unless they were already killed)

What I'm trying to say is the the attack value shouldn't update each time the character attacks someone else. It should only be updated once

I.e. if the attack value is 10, the damage value of the weapon is 5, the first iteration the attack value will be set to 15, the second time the character attacks someone else it will be set to 20